Lectures/code_review_example.R

```diff
@@ -0,0 +1,32 @@
+library(palmerpenguins)
+library(ggplot2)
+
+(penguins
+	%>% filter(island == "Torgersen")
+	%>% group_by(year, species) 
+	%>% summarise(count = n()) 
+	%>% ggplot(aes(x = year, y = count, color = species)) 
+	+ geom_line()
+	+ geom_point()
+	+ labs(title = "Species Count on Torgersen")
+)
+
+(penguins
+	%>% filter(island == "Biscoe")
```

Comment:
consider refactoring to generalize the code rather than hard-coding the island each time